src/styles/index.scss

```diff
@@ -1,3 +1,122 @@
 body {
+  font-family: "Roboto", sans-serif;
   margin: 0;
 }
+
+.card {
```

Comment:
you need to move card and other BEM blocks to separate files, please check the checklist:
[FILE STRUCTURE] - You should create separate file for each BEM block